src/main.rs

```diff
@@ -40,7 +40,11 @@ fn init(gfx: &mut Graphics) -> State {
 }
 
 fn texture(gfx: &mut Graphics, img: &[u8]) -> Texture {
-    gfx.create_texture().from_image(img).build().unwrap()
+    gfx.create_texture()
+        .from_image(img)
+        .with_premultiplied_alpha()
```

Comment:
this option helps to transform a non-premultiplied alpha texels from RGBA values to "a*R, a*G, a*B, a".